src/App.tsx

```diff
@@ -29,10 +29,7 @@ const DatabaseRoute: React.FC<{ match: match<{ namespace: string }> }> = ({
 const App: React.FC = () => {
   return (
     <div className="App">
-      <header className="App-header">
-        <h1>Firebase Emulator Suite</h1>
-        <Navigation />
-      </header>
+      <AppBar />
```

Comment:
While we're at it, can we also remove `src/components/Navigation/**`? If I read this correctly we don't need my legacy hacky navigation panel.

Reply:
Done.